lib/src/arithmetic/signals/fixed_point_logic.dart

```diff
@@ -123,6 +123,13 @@ class FixedPoint extends Logic {
     }
   }
 
+  /// Multiply
+  Logic fpMultiply(dynamic other) {
```

Comment:
I think you can either move this directly into the implementation of `operator *`, or at least make this private.  If you keep it as a separate function, why not just `_multiply` since it's already applicable to FixedPoint?

Reply:
This I will probably need some Dart education on. I like how we can use `.eq()` and `.lte()` methods or the override. That could just be my preference so not married to it. Would like to set it private though.

Reply:
Fixed to be _multiply